crates/iroha_telemetry_derive/src/lib.rs

```diff
@@ -256,34 +264,34 @@ fn impl_metrics(emitter: &mut Emitter, _specs: &MetricSpecs, func: &syn::ItemFn)
     };
 
     #[cfg(not(feature = "metric-instrumentation"))]
-    let res = quote!(
+    quote!(
         #(#attrs)* #vis #sig {
             #block
         }
-    );
-    res
+    )
 }
 
+// FIXME: metrics were removed https://github.com/hyperledger/iroha/issues/5134
 #[cfg(feature = "metric-instrumentation")]
 fn write_metrics(
     metric_arg_ident: proc_macro2::Ident,
     specs: MetricSpecs,
 ) -> (TokenStream, TokenStream, TokenStream) {
     let inc_metric = |spec: &MetricSpec, kind: &str| {
         quote!(
-            #metric_arg_ident
-                .metrics
-                .isi
-                .with_label_values( &[#spec, #kind ]).inc();
+            // #metric_arg_ident
+            //     .metrics
+            //     .isi
+            //     .with_label_values( &[#spec, #kind ]).inc();
```

Comment:
was this intended?

Reply:
if so, make it a ticket

Reply:
There is no `.metrics` field anymore. This macro is just broken, but is still used in many places. I decided to remove codegen from it rather than removing it from everywhere.

Reply:
There is already a ticket about this, https://github.com/hyperledger/iroha/issues/3910